release.xsh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env xonsh 
+"""Release helper script for xonsh."""
+import os
+import re
+import sys
+from argparse import ArgumentParser, Action
+
+def replace_in_file(pattern, new, fname):
+    """Replaces a given pattern in a file"""
+    with open(fname, 'r') as f:
+        raw = f.read()
+    lines = raw.splitlines()
+    ptn = re.compile(pattern)
+    for i, line in enumerate(lines):
+        if ptn.match(line):
+            lines[i] = new
+    upd = '\n'.join(lines) + '\n'
+    with open(fname, 'w') as f:
+        f.write(upd)
+
+
+def version_update(ver):
+    """Updates version strings in relevant files."""
+    pnfs = [
+        ('__version__\s*=.*', "__version__ = '{0}'".format(ver), 
+         ['xonsh', '__init__.py']),
+        ('version:\s*', 'version: {0}.{{build}}'.format(ver), '.appveyor.yml'),
```

Comment:
`'.appveryor.yml'` should be `['.appveryor.yml']` (wrapped in a list or tuple) so that it doesn't get expanded as `./a/p/p/v/e/y/o/r/./y/m/l`.